src/CoreWCF.NetTcp/src/CoreWCF/Channels/Framing/FramingEncoder.cs

```diff
@@ -130,7 +130,23 @@ private EncodedContentType(string contentType)
 
         public static EncodedContentType Create(string contentType)
         {
-            if (contentType == FramingEncodingString.BinarySession)
+            if (contentType == FramingEncodingString.ExtendedBinarySessionDeflate)
+            {
+                return new EncodedContentType(FramingEncodingType.ExtendedBinarySessionDeflate);
+            }
+            //else if (contentType == FramingEncodingString.ExtendedBinaryDeflate)
+            //{
```

Comment:
Same comment as previous, we need to non-session variations too.

Reply:
Uncommented these non-session variants as well